src/content/reference/react/Component.md

```diff
@@ -1271,6 +1271,16 @@ We recommend defining components as functions instead of classes. [See how to mi
 
 By default, if your application throws an error during rendering, React will remove its UI from the screen. To prevent this, you can wrap a part of your UI into an *Error Boundary*. An Error Boundary is a special component that lets you display some fallback UI instead of the part that crashed--for example, an error message.
 
+<Note>
+Error boundaries do not catch errors for:
+
+- Event handlers [(learn more)](/learn/responding-to-events)
+- Asynchronous code (e.g. `setTimeout` or `requestAnimationFrame` callbacks)
```

Comment:
Can we caveat that with `startTransition` from `useTransition` in those scenarios behaving different? Erroring in a Transition that is bound to the Hook version will trigger Error boundaries.